### PR TITLE
added pure number to token operator

### PR DESCRIPTION
**Pull request by SoluMilken**
#### Recognize string in a list and replace it with a special token (`_num_`) if the whole string are pure digits.

```python
        >>> from uttut.pipeline.ops.pure_num_to_token import PureNumtoToken
        >>> op = PureNumtoToken()
        >>> output_seq, output_labels, realigner = op.transform(
        ["I", "like", "10", "W9", "apples", "300g", "."])
        >>> output_labels = label_aligner.transform([1, 2, 3, 4, 5, 6, 7])
        >>> output_seq
        ["I", "like", "_num_", "W9", "apples", "300g", "."]
        >>> output_labels
        [1, 2, 3, 4, 5, 6, 7]
        >>> label_aligner.inverse_transform(output_labels)
        [1, 2, 3, 4, 5, 6, 7]
```

### CODE REVIEW

**Comment by stegben** [comment]
也許改名成 `PureNum2Token` ?

File: uttut/pipeline/ops/pure_num_to_token.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    def __init__(self, token: str = None):
+        """
+
+        Recognize string in a list and replace it with a special token (_num_)
+        if the whole string are pure digits.
+
+        E.g.
+        >>> from uttut.pipeline.ops.pure_num_to_token import PureNumtoToken
+        >>> op = PureNumtoToken()
```
Comment (stegben) [inline uttut/pipeline/ops/pure_num_to_token.py]: `PureNum2Token`

File: uttut/pipeline/ops/pure_num_to_token.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    _input_type = list
+    _output_type = list
+
+    REGEX_PATTERN = re.compile(r"[\d\uFF10-\uFF19]+")
+
+    def __init__(self, token: str = None):
+        """
+
+        Recognize string in a list and replace it with a special token (_num_)
```
Comment (absolutelyNoWarranty) [inline uttut/pipeline/ops/pure_num_to_token.py]: Given a list of strings, replace strings which have only numeric characters with a special token.

File: uttut/pipeline/ops/pure_num_to_token.py (hunk 1 above)
Comment (SoluMilken) [reply uttut/pipeline/ops/pure_num_to_token.py]: f1c5e47

File: uttut/pipeline/ops/pure_num_to_token.py (hunk 2 above)
Comment (SoluMilken) [reply uttut/pipeline/ops/pure_num_to_token.py]: f1c5e47

**Pull request merged**